src/Microsoft.AspNet.OData.Shared/DataModificationExceptionType.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.OData.Builder;
+
+namespace Org.OData.Core.V1
```

Comment:
The namespace is by design here?

Reply:
Yes, we need to have the same namespace name in order to consider it as vocabs in the core lib @mikepizzo 

---
In reply to: [604611580](https://github.com/OData/WebApi/pull/2455#discussion_r604611580) [](ancestors = 604611580)